Formula/swiftgen.rb

```diff
@@ -22,21 +22,21 @@ def install
     ENV["GEM_HOME"] = buildpath/"gem_home"
     system "gem", "install", "bundler"
     ENV.prepend_path "PATH", buildpath/"gem_home/bin"
-    system "bundle", "install"
+    system "bundle", "install", "--without", "development"
     system "bundle", "exec", "rake", "cli:install[#{bin},#{lib},#{pkgshare}/templates]"
 
     fixtures = {
-      "Resources/Fixtures/XCAssets/Images.xcassets" => "Images.xcassets",
-      "Resources/Fixtures/XCAssets/Colors.xcassets" => "Colors.xcassets",
-      "Resources/Fixtures/Colors/colors.xml" => "colors.xml",
-      "Resources/Fixtures/Strings/Localizable.strings" => "Localizable.strings",
-      "Resources/Fixtures/Storyboards-iOS" => "Storyboards-iOS",
-      "Resources/Fixtures/Fonts" => "Fonts",
-      "Resources/Tests/Expected/XCAssets/swift3-context-all.swift" => "xcassets.swift",
-      "Resources/Tests/Expected/Colors/swift3-context-defaults.swift" => "colors.swift",
-      "Resources/Tests/Expected/Strings/structured-swift3-context-localizable.swift" => "strings.swift",
-      "Resources/Tests/Expected/Storyboards-iOS/swift3-context-all.swift" => "storyboards.swift",
-      "Resources/Tests/Expected/Fonts/swift3-context-defaults.swift" => "fonts.swift",
+      "Tests/Fixtures/Resources/XCAssets/Images.xcassets" => "Images.xcassets",
+      "Tests/Fixtures/Resources/XCAssets/Colors.xcassets" => "Colors.xcassets",
+      "Tests/Fixtures/Resources/Colors/colors.xml" => "colors.xml",
+      "Tests/Fixtures/Resources/Strings/Localizable.strings" => "Localizable.strings",
+      "Tests/Fixtures/Resources/Storyboards-iOS" => "Storyboards-iOS",
+      "Tests/Fixtures/Resources/Fonts" => "Fonts",
+      "Tests/Fixtures/Generated/XCAssets/swift3-context-all.swift" => "xcassets.swift",
+      "Tests/Fixtures/Generated/Colors/swift3-context-defaults.swift" => "colors.swift",
+      "Tests/Fixtures/Generated/Strings/structured-swift3-context-localizable.swift" => "strings.swift",
+      "Tests/Fixtures/Generated/Storyboards-iOS/swift3-context-all.swift" => "storyboards.swift",
+      "Tests/Fixtures/Generated/Fonts/swift3-context-defaults.swift" => "fonts.swift",
```

Comment:
This sort of large diff of list of files to install shows why we need a more general install mechanism than is present currently. Maybe we could install in `pkgshare` all of `Tests/Fixtures`?

Reply:
All of Tests/Fixtures will be a lot, especially a lot of fixtures we have to tests storyboards, but also images (xcassets) & fonts. Those fixtures are not necessary at all for SwiftGen to work properly, and are only used by `brew test swiftgen`, nothing else, so seems like a shame to install all of `Tests/Fixtures` and unnecessary files and take space on the user's disk just for that, hence the cherry-picked list.